lib/dishes.rb

```diff
@@ -1,6 +1,7 @@
 class Dishes
+  attr_accessor :menu
 
-  def view
-    true
+  def view(menu)
+    @menu = menu
```

Comment:
good use of refactoring. I like how simple the test was to allow dishes to understand view and how you have now added additional functionality.